packages/@sanity/default-layout/src/update/changelog/ChangelogDialog.tsx

```diff
@@ -20,7 +20,8 @@ const StickyBox = styled(Box)`
 export function ChangelogDialog(props: ChangelogDialogProps) {
   const {changelog, currentVersion, dialogProps, latestVersion} = props
 
-  const hasChangelog = changelog.filter((c) => c?.changeItems).length > 0
+  const changelogWithChangeItems = (changelog || []).filter((c) => c?.changeItems)
```

Comment:
In theory, the changelog can have a `changeItems` property that _is_ an array, but is empty. I think it would make sense to check that it both exists and is not empty. Agreed?

```suggestion
  const changelogWithChangeItems = (changelog || []).filter((c) => c?.changeItems?.length > 0)
```

Reply:
That's a good idea! Agree 👍